opm/output/data/Aquifer.hpp

```diff
@@ -44,13 +44,13 @@ namespace Opm { namespace data {
     };
 
     struct AquiferData {
-        int aquiferID;    //< One-based ID, range 1..NANAQ
-        double pressure;  //< Aquifer pressure
-        double fluxRate; //< Aquifer influx rate (liquid aquifer)
+        int aquiferID = 0;    //< One-based ID, range 1..NANAQ
```

Comment:
These values are all of them nonsensical - is it necessary to initialize in this way?

Reply:
ID isn't all that important but the others are because i add up values from all processes, including root.

Reply:
IMHO, it helps if everything is initialized in all cases. If you use valgrind or some other memory debugging tool, it will complain if uninitialized values are written and sent. It is usually quite hard for developers not 100% familiar with source code section to judge whether a variable is sensical (don't even know this word, hope I used it correctly) or not.

Alternatively, we might just not send it all.